contract-testing/pact-messaging/provider/build.gradle.kts

```diff
@@ -0,0 +1,19 @@
+plugins {
+    id("org.springframework.boot")
+    id("io.spring.dependency-management")
+
+    kotlin("jvm")
+    kotlin("plugin.spring")
+}
+
+dependencies {
+    implementation("org.springframework.boot:spring-boot-starter-web")
+    implementation("com.fasterxml.jackson.module:jackson-module-kotlin")
+    implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8")
+    implementation("org.jetbrains.kotlin:kotlin-reflect")
+
+    testImplementation("au.com.dius.pact.provider:junit5spring:4.2.16")
```

Comment:
should be managed in root build script and should only be `au.com.dius.pact.provider:junit5`

Reply:
done